package-scripts.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
     dx: concurrent.nps('build', 'lint', 'meta.dependencies'),
     meta: {
       dependencies: {
-        build: `depcruise -c .dependency-cruiser.js -T dot components pages apiClient contexts data definitions hooks public styles utils --progress -x node_modules definitions | dot -T svg > dependency-graph.svg`,
+        build: `depcruise -c .dependency-cruiser.js -T dot components pages apiClient data definitions hooks public styles utils --progress -x node_modules definitions | dot -T svg > dependency-graph.svg`,
```

Comment:
Yeah, it was yelling because we removed the `contexts` folder. I'll try to keep things separate in the future, but I think this was the outcome of the last task of last year becoming a bigger thing.